src/ProjectService.Validation/Image/CreateImagesRequestValidator.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
+using LT.DigitalOffice.ProjectService.Validation.Image.Interfaces;
+
+namespace LT.DigitalOffice.ProjectService.Validation.Image
+{
+  public class CreateImagesRequestValidator : AbstractValidator<CreateImagesRequest>, ICreateImagesRequestValidator
+  {
+    public CreateImagesRequestValidator(
+      IImageValidator imageValidator,
+      IProjectRepository projectRepository)
+    {
+      RuleFor(images => images.Images)
+        .Cascade(CascadeMode.Stop)
+        .NotNull().WithMessage("List must not be null.")
```

Comment:
It's not entirely clear what's wrong
better "List of images must not be null."

Reply:
done